### DISCUSSION TITLE
Blacklist Issue

### DISCUSSION THREAD

**Discussion by djl314** [DNSCrypt/dnscrypt-proxy#2291] [Q&A] [answered]
question is related to using the blacklist python script w/ cron.

apologize as I'm a newbie :( - I'm trying to schedule the blacklist import and although the cron job runs, the actual script seems like it not working.  I can run the script manually

I figure it's a syntax issue related to the path or another dumb mistake - here is the cron job (tried both ways):

```
*/3 * * * * python3 /home/ubuntu/blacklist/generate.py
*/3 * * * * /home/ubuntu/blacklist python3 /.generate.py (also tried w/out the './')
```

running cron as user w/ admin priv - syslog shows the job runs on Ubuntu 22

thx
DL

**Comment by jedisct1** [comment, marked as answer]
Maybe you need to go to the proper directory first:

```
cd /home/ubuntu/blacklist && python3 /home/ubuntu/blacklist/generate.py
```
